Content.Server/Singularity/EntitySystems/ContainmentFieldGeneratorSystem.cs

```diff
@@ -65,7 +65,8 @@ public override void Update(float frameTime)
     /// </summary>
     private void HandleGeneratorCollide(Entity<ContainmentFieldGeneratorComponent> generator, ref StartCollideEvent args)
     {
-        if (_tags.HasTag(args.OtherEntity, generator.Comp.IDTag))
+        if (_tags.HasTag(args.OtherEntity, generator.Comp.IDTag) &&
```

Comment:
Flip the order of these checks for mini perf boost

Reply:
I doubt it'll shave off more than a few dozen nanoseconds in practice, but sure, let's go for the nano perf boost. Fixed.